src/Camp25US.elm

```diff
@@ -0,0 +1,341 @@
+module Camp25US exposing (..)
+
+import Camp24Uk.Archive
+import Camp24Uk.Artifacts
+import Element exposing (..)
+import Element.Font as Font
+import Html
+import Html.Attributes
+import MarkdownThemed
+import Route exposing (..)
+import Theme
+
+
+meta =
+    { logo = { src = "/elm-camp-tangram.webp", description = "The logo of Elm Camp, a tangram in green forest colors" }
+    , tag = "Michigan, US 2025"
+    , location = "🇺🇸 Watervliet, Michigan"
+    , dates = "Tues 24th - Fri 27th June 2025"
+    , artifactPicture = { src = "/24-colehayes/artifacts-mark-skipper.png", description = "A suitcase full of artifacts in the middle of a danish forest" }
+    }
+
+
+view model subpage =
+    Element.column
+        [ Element.width Element.fill, Element.height Element.fill ]
+        [ Element.column
+            (Element.padding 20 :: Theme.contentAttributes ++ [ Element.spacing 50 ])
+            [ Theme.rowToColumnWhen 700
+                model
+                [ Element.spacing 30, Element.centerX, Font.center ]
+                [ Element.image [ Element.width (Element.px 300) ] meta.artifactPicture
+                , Element.column [ Element.width Element.fill, Element.spacing 20 ]
+                    [ Element.paragraph [ Font.size 50, Font.center ] [ Element.text "Archive" ]
+                    , elmCampDenmarkTopLine
```

Comment:
Let's get rid of the Denmark from the name of this function. Maybe just change to `topLine`